resources/static-folder/src/lib.rs

```diff
@@ -20,6 +20,16 @@ pub enum Error {
     Copy(fs_extra::error::Error),
 }
 
+#[derive(Serialize, Deserialize)]
+pub struct Paths {
+    // Build storage, where service files including assets are downloaded into
+    input: PathBuf,
+    // The service storage, used at runtime
+    output: PathBuf,
```

Comment:
It would be nice to make the fields public. This would make it easy to extend or use the static folder. I was relying on the output being accessible to make [this library](https://github.com/robertohuertasm/shuttle-env-vars/blob/master/src/lib.rs#L98) work. Now it's not possible to construct the `Paths` struct.

Reply:
Ah, I see. I wasn't aware about this use-case. Does this mean 0.24.0 is breaking your app, since it relies on the library?

Reply:
It works, so no problem for now. Anyway, I still think it would be nice to change the api of this library to make it easier extending it.

Reply:
Yes, we'll probably want to expose just the output dir for now.

Reply:
The problem of only making one field public is that the struct cannot be created outside the original crate. I'm not suggesting to drastically change the API if it's not needed, but I'm wondering what would be the problem of exposing everything, given that it's not possible to access to that resource from the public API anyway.

Reply:
That's a good point, I am not sure if exposing everything will cause problems but we'll have to double-check.